src/Razor/src/Microsoft.VisualStudio.RazorExtension/EmbeddedGrammars/aspnetcorerazor.tmLanguage.tmTheme

```diff
@@ -9,15 +9,15 @@
 
       <!-- Begin General -->
 
-      <dict>
+      <!-- <dict>
         <key>scope</key>
         <string>string - meta.attribute, punctuation.definition.string - meta.attribute, punctuation.definition.template-expression, constant.character.escape, constant.regexp, markup.heading, string.quoted.double.cs</string>
```

Comment:
Hmm, this will also end up lobotomizing our JS colorization wont it? I wonder if we actually want to do this. Also, in our latest update we actually support C# comments which will then mean the TextMate grammar will properly colorize the bits as acomment since it wont attempt to use Razor's version of the comment. I wonder if we should just wait for the TExtMate override bug

Reply:
Yes it will. If we just want this to wait for https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1336412 that's fine with me, I'll close this PR and resolve the issue as external.

Reply:
Ya I feel like we should otherwise we'll risk wrecking JS/CSS